.github/workflows/publish-stable.yml

```
@@ -37,10 +37,10 @@ jobs:
     - name: Checkout SunrisePrivate
       uses: actions/checkout@v4.2.2
       with:
-        repository: ${{ secrets.SUNRISE_PRIVATE_REPO }}
+        repository: ${{ secrets.MAKURA_PRIVATE_REPO }}
         path: SunrisePrivate
-        ssh-key: ${{ secrets.SUNRISE_PRIVATE_REPO_SSH_KEY }}
-        ref: ${{ inputs.private_commit_hash }}
+        ssh-key: ${{ secrets.MAKURA_PRIVATE_REPO_SSH_KEY }}
+        ref: ${{ inputs.private_ref }}
```

actionlint (1.7.12)

[error] property "private_ref" is not defined in object type {private_commit_hash: string}

(expression)